pkg/sfu/downtrack.go

```diff
@@ -166,6 +177,13 @@ func (d *DownTrack) OnCloseHandler(fn func()) {
 	d.onCloseHandler = fn
 }
 
+func (d *DownTrack) OnBind(f func()) {
+	debounced := debounce.New(250 * time.Millisecond)
```

Comment:
why do we need to debounce here?

Reply:
Im fixing that now, to not use debouce, and just send reports for stream id